### DISCUSSION TITLE
use SineFromTable with set_ssid_callback

### DISCUSSION THREAD

**Discussion by StyxyDog** [pschatzmann/ESP32-A2DP#397] [Q&A]
Hi 
I do not have the capacity to understand your library in any real depth. But I have been running your example code and having great results.
What I would really like to do is to be able to use  example file [streams-generator_sinfromtable-audiokit.ino](https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-audiokit/streams-generator_sinfromtable-audiokit/streams-generator_sinfromtable-audiokit.ino) to create a sine wave that is changed based on external sensors. But with the ability to connect to an unknown but available Bluetooth sink as in [bt_music_sender_ssid_callback.ino](https://github.com/pschatzmann/ESP32-A2DP/blob/main/examples/bt_music_sender_ssid_callback/bt_music_sender_ssid_callback.ino).
I think I can make my project work without this ability of pairing to an unnamed Bluetooth speaker, but it would be nice to do it.
Could you tell me if it is possible and point me in the right direction?
Thank you for your efforts in creating this library and any assistance you can offer.

**Comment by pschatzmann** [comment]
Just don't specify any name!

**Reply by StyxyDog** [reply]
🤦thank you!
